tools/install.sh

```diff
@@ -188,35 +189,42 @@ ask_config() {
 }
 
 write_config_file() {
-    cat >${1} <<EOT
+    local config_file=${1:-${LOCAL_CONFIG_FILE:-default}}
+    if [ -z "$LOCAL_CONFIG_FILE" ]
+    then
+        cat >${1} <<EOT
 {
-  "core": {
+"core": {
     "metrics": {
-      "instance": "$HOSTNAME",
-      "comment": "$METRICS_COMMENT",
-      "push": "$PUSHMETRICS"
+    "instance": "$HOSTNAME",
+    "comment": "$METRICS_COMMENT",
+    "push": "$PUSHMETRICS"
     }
-  },
-  "spn": {
+},
+"spn": {
     "publicHub": {
-      "name": "$HOSTNAME"
+    "name": "$HOSTNAME"
     }
-  }
+}
 }
```

Comment:
Please don't change the indentation here.

Reply:
thanks for the feedback, see the new commit